src/pkg/file_preview/kitty.go

```
@@ -0,0 +1,170 @@
+package filepreview
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/BourgeoisBear/rasterm"
```

golangci-lint (1.64.8)

File is not properly formatted

(goimports)